osu.Game/Screens/Play/HUD/SkinnableComboCounter.cs

```diff
@@ -1,29 +1,16 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using osu.Framework.Bindables;
 using osu.Game.Skinning;
 
 namespace osu.Game.Screens.Play.HUD
 {
-    public class SkinnableComboCounter : SkinnableDrawable, IComboCounter
+    public class SkinnableComboCounter : SkinnableDrawable
```

Comment:
I left this class in place as it feels cleander than shoving the `base()` spec inline at usages.